### ISSUE TITLE
WP8 issues about the show method

### ISSUE THREAD

**Issue by bau720123** [Paldom/SpinnerDialog#10]
hi @Paldom 
I am using PGB 3.6.3 with all core plugin and plugin 1.1.0 version
https://build.phonegap.com/plugins/1202

according to this
https://github.com/Paldom/SpinnerDialog/blob/3487c44/README.md

situation 1
if I only use this show method

```
window.plugins.spinnerDialog.show(null, "message123");
setTimeout(function() { window.plugins.spinnerDialog.hide(); }, 3000);
```

the spinner will never ever show in WP8

situation 2
if I use this show method and the traditional javascript alert method，just like these

```
alert(123);
window.plugins.spinnerDialog.show(null, "message123");
setTimeout(function() { window.plugins.spinnerDialog.hide(); }, 3000);
```

when in Odd time，it will only show javascript alert
http://www.littlebau.com/20141103/01sinner.png
but...
when in Even time，it will show both all (the animation will show too)
http://www.littlebau.com/20141103/02sinner.png
In short，execute multiple times，you will know what I am talking about

PS
Android and iOS are all work great
